core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -245,6 +259,28 @@ protected void doStop() throws Exception {
     return server;
   }
 
+  static boolean enableBasicAuth(String authMethod) {
+    RestConfig.AUTHENTICATION_METHOD_VALIDATOR.ensureValid(RestConfig.AUTHENTICATION_METHOD_CONFIG, authMethod);
```

Comment:
Why do we do this here? Shouldn't this have already been validated by `ConfigDef`/`AbstractConfig` during parsing?